### ISSUE TITLE
Adding in alt text

### ISSUE THREAD

**Issue by ldecicco-USGS** [USGS-R/training-curriculum#36]
I figured out a way to do this:

Change the initial kniter chunk to:
```r
knit_hooks$set(plot=function(x, options) {
  sprintf("<img src='/%s%s-%d.%s'/ alt='/%s'/>",
          options$fig.path, options$label,
          options$fig.cur, options$fig.ext,
          options$fig.cap)

})

opts_chunk$set(
  echo=TRUE,
  fig.path="static/plotFlowConc/",
  fig.width = 7,
  fig.height = 7,
  fig.cap = "TODO"
)
```

Then, in each chunk that produces a figure, add `fig.cap` to the chunk options:

```
{r message=FALSE, fig.show='hold', out.width=c('300px', '300px'), fig.cap=c("ggplot2 Concentration Flow plot","EGRET Concentration Flow plot")}

```

**Comment by ldecicco-USGS** [comment]
Note I edited the comment a little later, in case you copied it right away. Check out the blog posts for examples.

**Comment by lindsaycarr** [comment]
Fixed by #45
